Sources/MapboxCoreNavigation/Feedback/EventsAPI.swift

```diff
@@ -13,11 +12,7 @@ extension EventsService: EventsAPI {
         sendTurnstileEvent(for: turnstileEvent)
     }
 
-    func sendQueuedEvent(with attributes: [String : Any]) {
+    func sendEvent(with attributes: [String : Any]) {
         sendEvent(for: Event(priority: .queued, attributes: attributes, deferredOptions: nil))
```

Comment:
I'm ok with this approach since 1Tap has some cases when events should be sent immediately.

Marking this API with `@_spi` is somewhat debatable but since NavSDK already has some APIs marked like this, we should be fine.